src/main/java/org/apereo/portlet/soffit/connector/UserDetailsService.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apereo.portlet.soffit.connector;
+
+import java.util.List;
+import java.util.Map;
+
+import org.apereo.portlet.soffit.model.v1_0.UserDetails;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+
+/**
+ * Responsible for issuing and parsing Bearer tokens.
+ *
+ * @author drewwills
+ */
+@Service
+public class UserDetailsService {
+
+    public static final String JWT_ISSUER = "Soffit";
```

Comment:
- [ ] Will be used by more than one `@Service`;  needs to be moved to a general location